src/features/HelpCompletion.ts

```diff
@@ -44,6 +44,10 @@ export class HelpCompletionFeature implements IFeature {
     }
 
     public onEvent(changeEvent: TextDocumentChangeEvent): void {
+        if (!changeEvent) {
+            return;
+        }
+
         this.helpCompletionProvider.updateState(
             changeEvent.document,
             changeEvent.contentChanges[0].text,
```

Comment:
Don't forget to undefined check `contentChanges[0]`.  If changeEvent itself was getting thru as undefined then we should have gotten an on `changeEvent.document` first.